components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/OIDCScopeHandler.java

```diff
@@ -49,15 +51,30 @@ public boolean validateScope(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
             // if id_token is allowed for requested grant type.
             return true;
         } else {
+
             // Remove openid scope from the token message context.
             String[] scopes = (String[]) ArrayUtils.removeElement(tokReqMsgCtx.getScope(), OAuthConstants.Scope.OPENID);
+
+            //Get all the registered oidc scopes of a tenant
```

Comment:
```suggestion
            // Get all the registered OIDC scopes of a tenant.
```
Follow the correct formatting for comments